src/main/java/com/monumental/services/UserService.java

```diff
@@ -45,6 +50,9 @@
     @Autowired
     VerificationTokenRepository tokenRepository;
 
+    @Autowired
+    ContributionRepository contributionRepository;
```

Comment:
Minor: In my PR I'm changing all of our `@Autowired` variables to be `private`. If you could make this one `private` that'd be awesome